### ISSUE TITLE
Support keyblox-like 1-key-finesse input

### ISSUE THREAD

**Issue by ImpleLee** [26F-Studio/Techmino#1188]
https://harddrop.com/wiki/KeyBlox

**Comment by ImpleLee** [comment]
In techmino, most actions are bound to a specific key - I mean, one can say those keys are the actual names of the functions. Probably this issue is quite hard?

**Comment by MrZ626** [comment]
yeah very hard

**Comment by SweetSea-ButImNotSweet** [comment]
Yeah except like, if I hook the entire game's mechanic to get that working
